polaris-react/src/components/Tabs/Tabs.tsx

```diff
@@ -603,13 +603,17 @@ export const Tabs = ({
                     disabled ? (
                       newTab
                     ) : (
-                      <Tooltip
-                        content={i18n.translate('Polaris.Tabs.newViewTooltip')}
-                        preferredPosition="above"
-                        hoverDelay={400}
-                      >
-                        {newTab}
-                      </Tooltip>
+                      <div>
```

Comment:
This is preventing the error but it look a like focus on the tablist is list completely based on 
https://storybook.polaris.shopify.com/?path=/story/all-components-tabs--with-actions
 Better than and error but tabs with action appear to have additional issues

Reply:
100 percent! This remove the errors and achieves parody. The next step is determining if tooltip should forwardRef its ref. Which _could_ be a breaking change 😢

Reply:
Perfect! This PR fixes that 😄 

https://github.com/Shopify/polaris/assets/24610840/c5899eec-7ddf-42a2-b789-f1fbecd81460